config/locales/models.pbs.de.yml

```diff
@@ -1189,6 +1190,9 @@ de:
       label_format:
         nickname: Pfadiname auf Etikett
 
+      service_token:
+        group_health: HealthCheck Export-Endpunkt
```

Comment:
Hätte ich umbenannt in HealthCheck-Daten HealthCheck Export-Endpunkt (Nur Opt-In)

Reply:
Kann man auch noch im Transifex machen, plus ich würde unbedingt einen HelpText über die weitreichenden Implikationen dieses Felds erfassen.

Reply:
Ich hab gedacht, das sollten an den bereits bestehenden Texten angepasst sein.
Den Helptext wollen wir auch machen, allerdings bei der Checkbox auf Abteilungsebene. Das Token sieht man ja nur auf Bundesebene.